src/bidiMapper/domains/network/networkProcessor.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Protocol from 'devtools-protocol';
+import {CdpClient} from '../../CdpConnection';
+import {IEventManager} from '../events/EventManager';
+
+import {NetworkRequest} from './networkRequest';
+
+export class NetworkProcessor {
+  readonly #eventManager: IEventManager;
+
+  readonly #requestMap = new Map<string, NetworkRequest>();
```

Comment:
Consider documenting this map (and use `/** */` when you do so).

Reply:
done